spec/dummy/config/application.rb

```diff
@@ -1,9 +1,12 @@
 # frozen_string_literal: true
 
 require 'rails'
+require 'activerecord-jdbc-adapter' if defined? JRUBY_VERSION
```

Comment:
fascinating

Reply:
I think this ended up being necessary. I couldn't reproduce many things locally.

Reply:
I feel that 🫂

Reply:
Yeah, it was necessary only for the latest version. There is a CI run below that you can see.